ftest.sh

```diff
@@ -320,8 +332,11 @@ fi
 
 # Remove the latest avocado symlink directory to avoid inclusion in the
 # jenkins build artifacts
-unlink $DAOS_BASE/src/tests/ftest/avocado/job-results/latest
-
+if [ $SL_PREFIX == "/usr" ]; then
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]